Tests/Foundation/Tests/TestDecimal.swift

```diff
@@ -808,6 +808,13 @@ class TestDecimal: XCTestCase {
         XCTAssertEqual(zero3.description, "0")
     }
 
+    func test_Significand() {
+        let x = -42 as Decimal
+        XCTAssertEqual(x.significand.sign, .plus)
+        let y = Decimal(sign: .plus, exponent: 0, significand: x)
+        XCTAssertEqual(y.sign, .minus)
```

Comment:
Could you expand the tests to explicitly test all 4 combinations of `sign`/`significand` being +/-, and test the result of the initialiser against the full value, thanks. I know its a bit cumbersome but I find with number tests its useful to cover all the cases if possible.